aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -184,6 +198,7 @@ module AptosFramework::Token {
         initial_balance: u64,
         maximum: u64,
         uri: vector<u8>,
+        royalty_basis_point: u64
```

Comment:
`royalty_basis_points`?

Reply:
all the reference updated to royalty_points.